lib/streamlit/elements/form.py

```diff
@@ -222,7 +222,7 @@ def form_submit_button(
         """
         return self.dg._button(
             label=label,
-            key=f"FormSubmitter:{current_form_id(self.dg)}",
+            key=f"FormSubmitter:{current_form_id(self.dg)}-{label}",
```

Comment:
It doesn't feel very readable to me. Can we split this into 2 lines?
```python
form_id = current_form_id(self.dg)
...
key=f"FormSubmitter:{form_id}-{label}",
```

Reply:
👍 done